# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

BallFollowingActionGoal action_goal
BallFollowingActionResult action_result
BallFollowingActionFeedback action_feedback
